### ISSUE TITLE
'resolve' command issues

### ISSUE THREAD

**Issue by Kiwi-i** [TheNexusAvenger/Nexus-Admin#17]
The `resolve` command in Cmdr has a conditional argument (provided as a function). With the way Nexus-Admin provides the command info, functions within the `Args` table will never be passed to the client (as per networking limitations). Thus the client always believes the `resolve` command only has 1 argument.

While there is an exception for the `Run/ClientRun` functions of commands, this exception is not used for arguments.

**Comment by Kiwi-i** [comment]
Possible (albeit hacky) solution to the problem is including the line
```lua
if ExistingCommand then CmdrCommandData.Args = ExistingCommand.Args end
```
Before the line
```lua
self.Cmdr.Registry:RegisterCommandObject(CmdrCommandData)
```
In the Nexus-Admin `ClientRegistry:LoadCommand()` method

This is definitely not a final solution but it solves the problem.

**Comment by TheNexusAvenger** [comment]
Resolved in #18.
